apps/dynatrace/perftest/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../dynatrace-operator2.yaml
```

Comment:
Can we clean up this naming now that these have been to moved to a separate namespace/folder - by removing that 2

Reply:
Done